src/hardware/vga_draw.cpp

```diff
@@ -1198,6 +1199,39 @@ static void maybe_aspect_correct_tall_modes(double &current_ratio)
 	current_ratio *= 2;
 }
 
+// Pre-VGA color modes often ran with a horizonal sync rate of 15.7 kHz
+// (200-lines) or 21.85 KHz (350-lines), both producing a vertical sync rate of
+// 60 Hz on those old systems, however newer VGA systems often run at a slightly
+// higher clock rate. So we adjust the clock when not running a pre-VGA system.
+// Ref: http://minuszerodegrees.net/video/bios_video_modes.htm
+static uint32_t get_clock_bias_for_pre_vga_color_mode(const uint32_t clock_hz,
```

Comment:
Hmm, I think these are generally referred to as "video modes", not "color modes". E.g. this is from the "The Assembly Language database" by Peter Norvig, from 1987:

http://vitaly_filatov.tripod.com/ng/asm/asm_023.1.html

Reply:
I'm using the word 'color' in the comments and variables (admittedly excessively!) to exclude the MDA mode(s) and Hercules machine type.

I'll state that more clearly in the comment, and then I'll drop the _colour_ chunk from these variables.

Reply:
Fixed.